nexusparser/tools/dataconverter/convert.py

```diff
@@ -0,0 +1,157 @@
+"""This script runs the conversion routine using a selected reader and write out a Nexus file."""
+
+import glob
+import importlib.machinery
+import importlib.util
+import json
+import logging
+import os
+import sys
+from typing import List, Tuple, Dict
+import xml.etree.ElementTree as ET
+
+import click
+
+from nexusparser.tools.dataconverter.writer import Writer
+
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+# Nexus related functions to be exported in a common place for all tools
```

Comment:
This comment is actually for both the functions below it. It's just to tag it for me. Maybe a TODO: was better here. I'll move these to the helper module we have setup for dealing with nexus related stuff. Therefore, this will be gone in a future commit.